fennel/featuresets/test_derived_extractors.py

```diff
@@ -57,7 +57,7 @@ class UserInfo:
             default="unspecified",
         )
         # lookup with meta
-        age_years: float = (
+        age_years: int = (
```

Comment:
In the dataset age is defined as a float so why this change?

Reply:
No its a float

Reply:
Oh sorry I was looking at the User dataset not UserInfo